geode-redis/src/integrationTest/java/org/apache/geode/redis/general/ShutdownIntegrationTest.java

```diff
@@ -61,11 +61,7 @@ public void shutdownIsDisabled_whenOnlySupportedCommandsAreAllowed() {
 
     // Unfortunately Jedis' shutdown() doesn't seem to throw a JedisDataException when the command
     // returns an error.
-    try {
-      jedis.shutdown();
-    } catch (JedisConnectionException ignore) {
```

Comment:
Will `jedis.shutdown()` no longer throw `JedisConnectionException`? Or if  `JedisConnectionException` is thrown, we just don't want to catch it?

Reply:
It was throwing this before, intermittently, because the server was actually shutting down intermittently. The point of the test was for shutdown to be a NOOP. But the flag the test sets to make it a noop did not work because the thread reading the flag is in another thread so it sometimes did not see that the command was unsupported. So now with the "volatile" on the boolean, it will always see the change this test makes.

Reply:
Also, this fix was already reviewed on another branch. I just wanted it to be isolated on its own PR instead of being part of another PR.